src/index.css

```diff
@@ -14,7 +14,16 @@ body {
   background: #fff;
 }
 
-.whole-body {
+button {
+  background-color: rgb(53, 128, 226);
+  color: #fff;
+}
+
+p {
+  font-size: 14px;
+}
+
```

Comment:
- [ ] Kindly follow the given font-family for all the text in your project as this is a requirement. Please note that the font-family used varies for most of the text